src/applications/personalization/rated-disabilities/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "appName": "Rated Disabilities",
+  "entryFile": "./rated-disabilities-entry.jsx",
+  "entryName": "disability/my-rated-disabilities",
+  "rootUrl": "/disability/my-rated-disabilities",
+  "template": {
+    "layout": "page-react-sidebar.html"
```

Comment:
If this page should be kept out of prod for X amount of time, you could feature flag it here by -

```
"template": {
    "layout": "page-react-sidebar.html",
    "vagovprod": false
}
```

This means that the HTML file (controlled by the "template") shouldn't go to prod, but should go to Staging and lower environments. You can also import this manifest.json in other application  (like the profile) to conditionally render links to this app. The JS bundle built from this manifest will continue to be built, but there won't be an HTML landing page for it to render.